rust-bot/src/commands/mod_group.rs

```diff
@@ -19,12 +19,41 @@ use serenity::{
 
 #[group]
 #[checks(Mod)]
-#[commands(create_cohort)]
+#[commands(add_role_to_users, create_cohort)]
 struct Mod;
 
 // Party Corgi - Mod Role Id = 639531892437286959
 const MOD_ROLE_ID: u64 = 639531892437286959;
 
+#[command]
+#[aliases("add_role", "add-role")]
+fn add_role_to_users(ctx: &mut Context, msg: &Message, mut args: Args) -> CommandResult {
+    if (args.len() as u32) < 2 {
```

Comment:
do we need this cast to do a comparison?

Reply:
args.len() returned as a `usize` and so I did need to do some sort of cast, but I wonder if I could change this maybe to something like `try_into().unwrap()` maybe instead?

Reply:
What would be the benefit of try_into? Usize is guaranteed to convert to u32, isn't it? I was under the impression that try_into was for when there was a chance for the cast to fail, which we don't really need to worry about here?

Reply:
Did the compiler throw the error for you? Because I think Chris's right and that you typically don't have to cast usize for comparing to a "magic" digit. ([Rust playground for reference](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=fn%20main()%20%7B%0A%20%20%20%20let%20x%3Ausize%20%3D%201%3B%0A%20%20%20%20%0A%20%20%20%20if%20x%20%3C%202%20%7B%0A%20%20%20%20%20%20%20%20print!(%22x%20is%20less%20than%202%22)%0A%20%20%20%20%7D%0A%7D))

Reply:
Super strange, before I wrote this, I recreated the error and it produced the same issue that made me need to write it and now I lost it lmao 🙃  I wonder what I did before that made me need to go through this path because at the time it was a compiler error.

Reply:
I'm just gonna change it and say yolo